autogen/main/versions.tf.tmpl

```diff
@@ -17,6 +17,10 @@
 {% set module_path_str =  module_path|string %}
 {% set module_registry_name = module_path_str.split('/')[-1] %}
 
+locals {
+  blueprint_name = join("/", compact([var._parent_module, "terraform-google-kubernetes-engine{% if module_registry_name %}:{{ module_registry_name }}{% endif %}/v13.0.0"]))
```

Comment:
IIUC this is only used by the `private-cluster*` modules to attribute usage as `safer-cluster*` when used by `safer-cluster*` modules. Do we need to expose var._parent_module in all the gke modules?

Reply:
@bharathkkb We only use it for some for now, but my intention is to expose this consistently so anyone building higher-level blueprints can also pass in the info. This should become our standard pattern.